site/assets/scss/_content.scss

```diff
@@ -64,6 +64,14 @@
   min-width: 280px;
 }
 
+.table-swatches {
+  // Boosted mod: no `th`
+
+  td code {
```

Comment:
Not sure about that but should we remove this selectors and make it the same for every table in our docs ?